### PR TITLE
giftshop-userService

### PR DESCRIPTION
**Pull request by yeswanth1570**

### CODE REVIEW

File: giftshop/service/user-service.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+
+const dbUserName = 'apikey-v2-2djdlrrbf736ap4aa6rlre2x1j1wf65v1ti1e8x2bihn';
+const dbPassword = '3bc2893c0a2a1ec42d9b17840b18447b';
+var endpoint = 'https://75b0afe3-3fa7-477b-8352-bdcfcd522a16-bluemix.cloudantnosqldb.appdomain.cloud/';
+const basicAuth = 'Basic ' + btoa(dbUserName + ':' + dbPassword);
+
+
+class crud {
```
Comment (hariharansa) [inline giftshop/service/user-service.js]: rename class

Comment (yeswanth1570) [reply giftshop/service/user-service.js]: ok

Comment (yeswanth1570) [reply giftshop/service/user-service.js]: changes completed

**Review by amri98 — COMMENTED** [review]
approved

File: giftshop/service/user-service.js (hunk 1 above)
Comment (hariharansa) [reply giftshop/service/user-service.js]: approved

**Review by hariharansa — APPROVED** [review]
approved

**Pull request merged**